frontend/lib/src/theme/globalStyles.ts

```diff
@@ -179,7 +179,6 @@ export const globalStyles = (theme: EmotionTheme): SerializedStyles => css`
   h1 {
     font-family: ${theme.genericFonts.headingFont};
     font-weight: ${theme.fontWeights.extrabold};
-    color: ${theme.colors.headingColor};
```

Comment:
Benny & I talked about cleaning up the globalStyles in the context of the advanced theming project today. A lot of these global styles feel like historical tech debt. It would be a lot cleaner if only the markdown component would overwrite the heading styles. Probably something we could tackle soon.

cc @raethlein